### ISSUE TITLE
Dark Theme displays incorrectly with "Increase Contrast" enabled

### ISSUE THREAD

**Issue by Sappharad** [Sappharad/mp3gainOSX#20]
Turning on the "Increase Contrast" option in Accessibility settings causes the grid text color to be black, resulting in black text on a black background that you can't read.

This seems like a bug with macOS dark vibrancy layers, but I'll see if I can work around it. Might also need to file a radar with Apple for them to fix it if it does turn out to be an OS problem.

**Comment by Sappharad** [comment]
I produced a stand-alone project that uses Vibrancy layers "correctly" and it doesn't have this problem. This tells me that the hack I implemented to apply dark mode automatically is probably causing the problem. Messing with the settings on the vibrancy layer doesn't seem to help.

The user who reported the bug didn't realize they had the increase contrast option enabled and was satisfied with just turning the option off. I want to fix this though, even if it means just disabling dark mode for high contrast. I will continue to play around with this at a later time.
